source/_imports/objects/o-offset-content.css

```diff
@@ -24,10 +32,14 @@ Styleguide Objects.OffsetContent
 		margin-right: var(--buffer);
 	}
 	.o-offset-content--left + .o-offset-content--left { clear: left; }
+
+  .figure:is(:--o-offset-content) .figure-caption {
+    padding-right: 125px;
+  }
 }
 @media only screen and (--medium-and-above) and (--max-wide-and-below) {
-  [class*="o-offset-content--"] {
-    max-width: 50%;
+  :--o-offset-content {
+    max-width: 560px;
   }
```

Comment:
<details><summary>The offset content changes support the use of `c-offset-content--right` on many blog posts.</summary>

- https://prod.ecep.tacc.utexas.edu/news/2021/02/25/cape-framework-published-communications-acm/
- https://prod.ecep.tacc.utexas.edu/news/2019/09/13/2019-state-computer-science-education-report-released/
- https://prod.ecep.tacc.utexas.edu/news/2017/11/15/ecep-alliance-shares-commitment-csforall-summit/
- https://prod.ecep.tacc.utexas.edu/news/2017/08/31/puerto-rico-set-teach-exploring-computer-science-spanish/
- https://prod.ecep.tacc.utexas.edu/news/2017/07/26/ecep-website-redesign-focus-access-content-and-design/
- https://prod.ecep.tacc.utexas.edu/news/2017/05/04/nv-nh-nc-pr-awarded-mini-grants-further-cs-education-equity/
- https://prod.ecep.tacc.utexas.edu/news/2017/02/01/rethinking-perkins-expand-access-k-12-computer-science/

</details>